lib/csrfToken.test.ts

```diff
@@ -0,0 +1,109 @@
+import { CSRFValidationError, init } from './csrfToken';
```

Comment:
Does this need to be a `.spec.ts` file? I'm not sure if `.test.ts` files are picked up?

Reply:
Refactored :)